### PR TITLE
Remove lookup table functionality

### PR DESCRIPTION
**Pull request by peterdesmet**
## Rationale

The lookup table (a csv file with that contains the mapping) was an interesting idea, but I think we should abandon it for this project and do all remapping in the code:

* That way we only have to update one file (the code).
* We always see the mapping in context: there is no need to read and display the values from the lookup table with `stack()`.
* Mapping can be more flexible: e.g. stripping out `?` before needs to be done in code anyway.
* We don't need a `term_mapping.R` function to select a term and handle edge cases.
* The `recode()` functionality provides parameters for missing or default values.
* We're not using the lookup table outside the project (so might as well have it in the code)
* You're more inclined to run the code if you adapt the mapping (as it's part of the code)
* The mapping is easy to read for non-codes in the [dwc_mapping.md file](https://github.com/trias-project/alienplantsbelgium/blob/b9a5e7417411d4ae37a7625553ebbfd8980c598e/src/dwc_mapping.md#occurrencestatus).

Basically, this:

```r
distribution %<>% mutate(occurrenceStatus = recode(presence_be,
  "X" = "present",
  "?" = "presence uncertain",
  .default = "",
  .missing = "absent"
))
```

Is easier to understand than this:

```r
occurrencestatus_lookup <- term_mapping(lookup_table, "occurrenceStatus")
stack(occurrencestatus_lookup)

distribution %<>% mutate(occurrenceStatus = 
  recode(presence_be, !!!occurrencestatus_lookup)
)
```

## Included in this pull request

* Use simple recode for occurrenceStatus (last term that used lookup table)
* Remove lookup table
* Remove term_mapping.R function
* Update repo structure in README

### CODE REVIEW

**Comment by qgroom** [comment]
I'm just trying to catch up where we where, but I seem to remember the lookup was to ensure that Filip knew what the choices were, rather than anything downstream.

**Comment by peterdesmet** [comment]
Are you talking about the lookup in Excel or lookup files in GitHub? I'm talking about the latter: want to make them part of the code raher than separate files.

**Comment by qgroom** [comment]
OK, I was thinking about the Excel lookups
q

**Comment by peterdesmet** [comment]
Thought so. So I assume you have no issues with this pull request (bundling all logic and code together in one file)?

**Comment by qgroom** [comment]
No, go crazy with it :-)
